Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3.mo

```diff
@@ -1251,10 +1290,10 @@ a model of the electrical motor and a continuous-time cascade controller.
         annotation (Dialog(group="Motor"));
       parameter Real ratio=-105 "Gear ratio" annotation (Dialog(group="Gear"));
       parameter SI.Torque Rv0=0.4
-        "Viscous friction torque at zero velocity in [Nm]"
+        "Viscous friction torque at zero velocity"
         annotation (Dialog(group="Gear"));
       parameter Real Rv1(unit="N.m.s/rad") = (0.13/160)
-        "Viscous friction coefficient in [Nms/rad]"
```

Comment:
Also change to SI.RotationalDampingConstant?

Reply:
What about SI.RotationalDampingConstant?

Reply:
Considering measurement units only, yes. But I'm not sure about proper physical meaning of the variable Rv1.
If it is common that only the unit itself is decisive, I have really no problem to change it.

Reply:
I see. It is consistent with the other occurrence of Rv1 in line 810. So ok.